src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -0,0 +1,180 @@
+# Contains the list of files to be excluded from license scanning.
+#
+# This list is processed using FileSystemName.MatchesSimpleExpression
+#
+# Format:
+#   Exclude the file entirely from license scanning: 
+#     <vmr-file-path>
+#   Exclude a specific detected license expression from the scan results for the file:
+#     <vmr-file-path>|<license-expression>[,<license-expression>...]
+
+#
+# aspnetcore
+#
+
+# Line 1 is a generic statement about license applicability that is being detected as "unknown"
+src/aspnetcore/src/Components/THIRD-PARTY-NOTICES.txt|unknown
+
+# Windows installer files that have a reference to a URL for license
+src/aspnetcore/src/Installers/Windows/**/*.wxl|unknown-license-reference
+src/aspnetcore/src/Installers/Windows/**/*.wxs|unknown-license-reference
+
+# License reference used in configuration, but not applying to code
+src/aspnetcore/src/Mvc/Settings.StyleCop|unknown-license-reference
+src/aspnetcore/src/submodules/MessagePack-CSharp/stylecop.json|unknown
+
+#
+# fsharp
+#
+
+# False positive
+src/fsharp/tests/EndToEndBuildTests/ProvidedTypes/ProvidedTypes.fs|unknown-license-reference
```

Comment:
Do you think we should file an issue to track submitting these "unknown-license-reference" to scancode?